FlowCrypt/Controllers/Setup/SetupViewController.swift

```diff
@@ -171,21 +179,7 @@ extension SetupViewController {
             }
     }
 
-    private func fetchEnctyptedKeys(with backupData: Data) {
```

Comment:
Was this method unused?

Reply:
this functionality moved to `BackupService` which is now fully responsible to fetch backups from provider(Map or Gmail) and parse keys. 
see `func fetchBackups(for userId: UserId) -> Promise<[KeyDetails]>` inside `BackupService`

just forgot to remove it from view controller